ext/dom/tests/DOMXPath_quote.phpt

```diff
@@ -0,0 +1,88 @@
+--TEST--
+Test DOMXPath::quote with various inputs
+--EXTENSIONS--
+dom
+--SKIPIF--
+<?php if (!class_exists('DOMXPath')) die('skip DOMXPath not available.'); ?>
+--FILE--
+<?php
+$dom = new DOMDocument();
+$xpath = new DOMXPath($dom);
+
+
+/**
+ * Quote a string for use in an XPath expression.
+ * 
+ * Example: $xp->query("//span[contains(text()," . $xp->quote($string) . ")]")
+ * 
+ * @param string $string string to quote.
+ * @return string quoted string.
+ */
+function UserlandDOMXPathQuote(string $string): string
```

Comment:
I wonder if it makes sense trying to add this to the fuzzer to see if it finds differences in behaviour?

Reply:
We could fuzz this, but it will be a bit cumbersome to write a specific driver for this. We could ask ourselves whether we want to do it generalised for all these kinds of quoting methods.